templates/AddressMap.ss

```diff
@@ -1,5 +1,5 @@
 <div class="addressMap">
 	<a href="//maps.google.com/?q=$Address">
-		<img src="//maps.googleapis.com/maps/api/staticmap?size={$Width}x{$Height}&markers=$Address&sensor=false" alt="$FullAddress.ATT" />
+		<img src="//maps.googleapis.com/maps/api/staticmap?size={$Width}x{$Height}&scale={$Scale}&maptype={$MapType}&markers=$Address&key=$Key" alt="$FullAddress.ATT" />
```

Comment:
Where does `$MapType` come from, and should there be an `if` statement around this? Also, with this and sensor removed, what version bump do you consider this having in relation to semantic versioning? Will this change break anything for users who are already using a static map from this module?

Reply:
I've removed $MapType.

Sensor has been removed: https://developers.google.com/maps/documentation/static-maps/intro#Moreinfo

I've tested this on existing sites and everything works correctly.